packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1619,4 +1619,54 @@ void main() {
     expect((mergeableMaterial.children.first as MaterialSlice).color, firstPanelColor);
     expect((mergeableMaterial.children.last as MaterialSlice).color, secondPanelColor);
   });
+
+  testWidgets('default materialGapSize should be 16', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: SingleChildScrollView(
+        child: ExpansionPanelList(
+          children: <ExpansionPanel>[
+            ExpansionPanel(
+              canTapOnHeader: true,
+              body: const SizedBox.shrink(),
+              headerBuilder: (BuildContext context, bool isExpanded) {
+                return const SizedBox.shrink();
+              },
+            )
+          ],
+        ),
+      ),
+    ));
+
+    final ExpansionPanelList expansionPanelList = tester.widget(find.byType(ExpansionPanelList));
+    expect(expansionPanelList.materialGapSize, 16);
+  });
+
+  testWidgets('ExpansionPanelList respects materialGapSize', (WidgetTester tester) async {
```

Comment:
i don't think this test holds much value, as you're just testing if the value you passed to the immediate argument is set correctly. it would be better to instead test if the effective gap b/w the items is correctly configured by `materialGapSize`.

after a short dive, i can see that `ExpansionPanelList` uses `MergeableMaterial` which adds `SizedBox` to a `ListBody` (which is like a `ListView`) for every `MaterialGap`. so you could test if the dimensions of the `SizedBox` are as per the value of `materialGapSize`.

Reply:
Yes you're right. I was having difficulties coming up with a good test. I'll come up with a good test. Thanks a lot!

Reply:
Added a better widget-test 🤞. Please have a look.

Reply:
oh this is great! thanks.

Reply:
I was unsure whether this new test was good or not.